kernelci.org/content/en/blog/posts/2021/03/16/index.html

```diff
@@ -97,4 +97,4 @@ <h3>Focus areas</h3>
 <p>Please keep in touch with what we’re up to or to get involved, you can read
 <a href="https://foundation.kernelci.org/blog/">our blog</a>, follow on twitter
 <a href="https://twitter.com/kernelci">@kernelci</a> or join our
-<a href="https://groups.io/g/kernelci/topics">mailing list</a>.</p>
+<a href="https://lore.kernel.org/kernelci/">mailing list</a>.</p>
```

Comment:
Verified OK on staging:
https://static.staging.kernelci.org/blog/posts/2021/03/16/

Reply:
There isn't one, that's a problem with the kernel dev lists.  You can go to this page:
https://subspace.kernel.org/lists.linux.dev.html

and then click on a link to subscribe which opens your email client i.e. `kernelci+subscribe@lists.linux.dev`.  I guess we could mention that somewhere but it's not as straightforward as a sing-up page like for groups.io.  Probably, having a section on kernelci.org explaining how to join the mailing list we can point to would help in cases like this.

Reply:
Yea not idea.

For this MR though you'd done what makes most sense. We can't have a massive explainer in a blog post about how to sign up.

Thanks